lib/widgets/month_expense.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:expense_tracker/models/expense.dart';
+import 'package:flutter/material.dart';
+
+/// Displays the total expenses of the current and last month.
+/// Is Stateless since it does not change.
+/// Is not visible if there are no expenses.
+class MonthlyExpenseWidget extends StatelessWidget {
+  const MonthlyExpenseWidget({super.key, required this.expenses});
+
+  final List<Expense> expenses;
+
+  /// Returns the total expenses of the given month.
+  double getTotalExpensesOfMonth(int month) {
+    var monthlyExpenses =
+        expenses.where((element) => element.date.month == month).toList();
```

Comment:
What if there is an expense from over a year ago? If `date.month` returns an int from 1 to 12, then this may lead to an issue